test/checks/design/InterfaceIsTypeCheckTest.hx

```diff
@@ -0,0 +1,40 @@
+package checks.design;
+
+import checkstyle.checks.design.InterfaceIsTypeCheck;
+
+class InterfaceIsTypeCheckTest extends CheckTestCase<InterfaceIsTypeCheckTests> {
+
+	public function testWithJustVars() {
+		assertMsg(new InterfaceIsTypeCheck(), InterfaceIsTypeCheckTests.TEST1, "Interfaces should describe a type and hence have methods");
```

Comment:
Specifying the type name here (`InterfaceIsTypeCheckTests.TEST1`) is unnecessary - can just be `TEST1`.